yt/yt/server/node/exec_node/gpu_manager.cpp

```diff
@@ -332,6 +335,13 @@ void TGpuManager::OnHealthCheck()
             }
 
             for (auto& gpuInfo : gpuInfos) {
+                if (!GpuDeviceIndices_.contains(gpuInfo.Index)) {
+                    YT_LOG_WARNING("Found unknown GPU device (DeviceName: %v)",
+                        GetGpuDeviceName(gpuInfo.Index));
```

Comment:
I'd prefer logging device index instead of using `GetGpuDeviceName`, but I understand we already do the latter for lost devices, so no call to action here.

Ideally, we should revise device identification and use device index (aka minor number) or UUID everywhere.

Reply:
Will do it in a separate PR!